net/curl/Portfile

```diff
@@ -352,10 +352,110 @@ subport curl-ca-bundle {
         set ca_bundle_dir ${prefix}/share/curl
         set openssl_dir ${prefix}/etc/openssl
         xinstall -d ${destroot}${ca_bundle_dir} ${destroot}${openssl_dir}
-        xinstall -m 644 ${worksrcpath}/lib/ca-bundle.crt ${destroot}${ca_bundle_dir}/curl-ca-bundle.crt
+        foreach dst [list mozilla-ca-bundle.crt curl-ca-bundle.crt] {
+            xinstall -m 644 ${worksrcpath}/lib/ca-bundle.crt ${destroot}${ca_bundle_dir}/${dst}
+        }
         ln -s ${ca_bundle_dir}/curl-ca-bundle.crt ${destroot}${openssl_dir}/cert.pem
     }
 
+    post-activate {
+        ui_info "Regenerating CA certificate bundle from keychain, this may take a while..."
+
+        set keychains {
+            /Library/Keychains/System.keychain
+            /System/Library/Keychains/SystemRootCertificates.keychain
+        }
+
+        ui_debug "Fetch certificates from keychains"
+        set certs_list [exec /usr/bin/security find-certificate -a -p {*}$keychains]
+        set certs [regexp -inline -all -- {-----BEGIN CERTIFICATE-----.*?-----END CERTIFICATE-----} $certs_list]
+
+        ui_debug "Filter valid certificates: not expired, SSL root"
+        set checkend_cmd "/usr/bin/openssl x509 -inform pem -checkend 0 -noout"
```

Comment:
I cannot check right now, but this _may_ fail on older macOS.
`curl` is a fundamental port, so this should be confirmed prior to merging.